### ISSUE TITLE
Need to use the H5P image library

### ISSUE THREAD

**Issue by tjb328** [h5p/h5p-image-hotspots#5]
The Image Hotspots do not use the H5P image library and are missing features such as ALT tags which are needed for accessibility.
